h2o-py/h2o/estimators/infogram.py

```diff
@@ -700,6 +700,23 @@ def algorithm_params(self):
         Customized parameters for the machine learning algorithm specified in the algorithm parameter.
 
         Type: ``dict``.
+
+        :examples:
+
+        >>> import h2o
+        >>> from h2o.estimators.infogram import H2OInfogram
+        >>> h2o.init()
+        >>> f = "https://erin-data.s3.amazonaws.com/admissible/data/taiwan_credit_card_uci.csv"
+        >>> col_types = {'SEX': "enum", 'MARRIAGE': "enum", 'default_payment_next_month': "enum"}
+        >>> df = h2o.import_file(path=f, col_types=col_types)
+        >>> train, test = df.split_frame(seed=1)
+        >>> y = "default_payment_next_month"
+        >>> x = train.columns
+        >>> x.remove(y)
+        >>> pcols = ["SEX", "MARRIAGE", "AGE"]
+        >>> ig = H2OInfogram(protected_columns=pcols)
+        >>> ig.train(y=y, x=x, training_frame=train)
+        >>> ig.plot()
```

Comment:
If it is example for `algorithm_params` I would expect call of `ig.algorithm_params`

Reply:
cc: @hannah-tillman

Reply:
Sorry, I was wrong here. Its a example for data_fraction so I would expect. to call data_fractions.. 

So just call: ig.data_fraction